libbpf-rs/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 - Added `ProgramOutput::duration` field which represent the average
   duration per repetition
 - Added `RingBuffer::consume_raw_n` method to consume up to N items
+- Added `target_obj_id` and `target_btf_id` fields to `TracingLinkInfo`
+  to expose BTF information directly from kernel queries
```

Comment:
Please move into a new `Unreleased` section. See https://github.com/libbpf/libbpf-rs/commit/af0910ec7a38cc2468e940700a9c66e5b990655f#diff-4c0b9d3ba9467bbce23979f2823a77684818168c500364d9652b70eac9dfe1f9

Reply:
Done in https://github.com/libbpf/libbpf-rs/compare/bae77fc949a8f468d5b57749882e28f6f77c1dc1..c4b8e11dc83fb80cf4ba8d3c8cf98ad6cad17d78.